roles/tas_single_node/templates/manifests/rekor/backfill_redis.sh.j2

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
+
+LAST_INDEX_FILE="/tmp/rekor_last_index"
+
+endIndex=$(curl -sS http://localhost:3001/api/v1/log | jq -r '.treeSize - 1')
+if [ "$endIndex" -lt 0 ]; then
+  echo "info: no rekor entries found"
+  exit 0
+fi
+
+if [ -f "$LAST_INDEX_FILE" ]; then
+  startIndex=$(cat "$LAST_INDEX_FILE")
+else
+  startIndex=0
+fi
+
+if [ "$startIndex" -gt "$endIndex" ]; then
+  startIndex=0
+fi
+
+if [ "$startIndex" -le "$endIndex" ]; then
+  podman run --rm --name backfill_redis \
+    -v "{{ tas_single_node_certs_dir }}/ingress-rekor.pem:/usr/local/share/ca-certificates/ingress-rekor.pem:ro" \
+    -e SSL_CERT_FILE=/usr/local/share/ca-certificates/ingress-rekor.pem \
+    --add-host="rekor.{{ tas_single_node_base_hostname }}:{{ dns_resolver }}" \
```

Comment:
I don't think the dns_resolver/add-host is necessary at all. All pods on the same network (use --network) should be able to connect to each other - see e.g. the rekor-server.j2 to see how it connects to trillian and Redis.

Reply:
I did try this first, I didn't have much luck if I am honest, backfill redis is awkward when connecting to the rekor instance it has to be a url so it can query the /api/v1/log/entries?logIndex=x api endpoint 

I will take another look, also when I think about it the unable to connect may have been related to the issue fixed in https://github.com/securesign/artifact-signer-ansible/pull/100, either way I will take a look